lib/Mail/DMARC/Report/Send/SMTP.pm

```diff
@@ -11,6 +11,10 @@ use Net::SMTPS;
 use Sys::Hostname;
 use POSIX;
 
+# Set strict mode for Email::MIME::ContentType
+# This prevents foo= and foo*0= being set at the same time.
+use Email::MIME::ContentType;
+$Email::MIME::ContentType::PRE_2231_FORM = 0;
```

Comment:
Do we want, instead, to set this locally just when generating the messages we generate?